### PR TITLE
[Merged by Bors] - chore(SetTheory/Cardinal/Basic): `mem_range_of_le_lift` → `mem_range_lift_of_le`

### PR DESCRIPTION
**Pull request by vihdzp**
This was a typo I made in #16958 only a week ago or so, so hopefully this doesn't warrant deprecations.

---

Admittedly the "incorrect" name also makes sense, but I think it's clearer to explicitly specify the function whose range we're talking about.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by YaelDillies — APPROVED** [review]
maintainer merge

**Comment by fpvandoorn** [comment]
Delegating out of fear to conflict with #17517 (maybe it won't). Feel free to merge after that is merged.

bors d+

**Pull request closed without merging**
